<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SshTunnelConfigDialog</class>
 <widget class="QDialog" name="SshTunnelConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SSH Tunnel Configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="remoteHostEdit">
     <property name="placeholderText">
      <string>Remote Host</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="remotePortEdit">
     <property name="placeholderText">
      <string>Remote Port</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="localPortEdit">
     <property name="placeholderText">
      <string>Local Port</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="usernameEdit">
     <property name="placeholderText">
      <string>Username</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="sshPortEdit">
     <property name="placeholderText">
      <string>SSH Port</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="startButton">
     <property name="text">
      <string>Start SSH Tunnel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="restartButton">
     <property name="text">
      <string>Restart SSH Tunnel</string>
     </property>
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>